cfy_manager/config.py

```diff
@@ -61,15 +61,18 @@ def dict_merge(dct, merge_dct):
 class Config(CommentedMap):
     TEMP_PATHS = 'temp_paths_to_remove'
 
-    def _get_installed_services(self):
+    @staticmethod
```

Comment:
If it's static it probably doesn't need to live in the class.

Reply:
Right, took it out to be in `common.py` as it will probably be in use by other functions.

Reply:
So it cannot sit in `common.py` due to circular imports. I moved it back to `config.py` but out of the class.